README.md

```diff
@@ -1,10 +1,9 @@
 # virt-who
 
-[![Build Status](https://travis-ci.org/candlepin/virt-who.svg?branch=main)](https://travis-ci.org/candlepin/virt-who)
-
 virt-who is agent for reporting virtual guest IDs to subscription manager.
 
-It works either locally or via remote hypervisor. It obtains association between hosts and guests in the given environment (e.g. hypervisor or local system) and then reports it to the subscription manager (SAM, Satellite 5, Satellite 6).
+It works either locally or via remote hypervisor. It obtains association between hosts and guests in the given
+environment (e.g. hypervisor or local system) and then reports it to the subscription manager.
```

Comment:
maybe replace "subscription manager" to "subscription management platform", so it covers both RHSM and Satellite?